silkworm/silkrpc/core/account_walker.cpp

```diff
@@ -110,13 +110,13 @@ Task<ethdb::SplittedKeyValue> AccountWalker::next(ethdb::SplitCursor& cursor, ui
     co_return skv;
 }
 
-Task<ethdb::SplittedKeyValue> AccountWalker::seek(ethdb::SplitCursor& cursor, uint64_t number) {
+Task<ethdb::SplittedKeyValue> AccountWalker::seek(ethdb::SplitCursor& cursor, BlockNum number) {
```

Comment:
The same change should be applied also in `account_walker.hpp`

Reply:
ok